### DISCUSSION TITLE
Can't get it work w-full

### DISCUSSION THREAD

**Discussion by sonmezcem** [tailwindlabs/tailwindcss#13503] [Help] [answered]
Hi folks,

I cannot get w-full work on my code. I am newbie that's the reason why I cannot fix it maybe someone can hit the box. I stated "I want this area use the full witdh of the parent element." Thank you.

```
<div class="block pt-1 pb-1">
<div class="max-w-screen-xl mx-auto flex flex-row bg-acikmavi-0 justify-center text-beyaz-0 font-lato">

<div id="default-tab-content">
    <div class="hidden p-4 rounded-lg bg-acikmavi-0 dark:bg-gray-800" id="profile" role="tabpanel" aria-labelledby="profile-tab">
        I want this area use the full witdh of the parent element
    </div>
    <div class="hidden p-4 rounded-lg bg-acikmavi-0 dark:bg-gray-800" id="dashboard" role="tabpanel" aria-labelledby="dashboard-tab">
        I want this area use the full witdh of the parent element
    </div>
    <div class="hidden p-4 rounded-lg bg-acikmavi-0 dark:bg-gray-800" id="settings" role="tabpanel" aria-labelledby="settings-tab">
        I want this area use the full witdh of the parent element
    </div>
</div>
</div>
</div>

```

**Comment by alex-krasikau** [comment]
Hey @sonmezcem!

There's no `w-full` in the code you shared 😅.

Could you please create a reproduction example on https://play.tailwindcss.com/ ? We will help you with it.

**Reply by sonmezcem** [reply]
Hey Alex! 

Thank you for warning! I removed it while trying. Now, I will provide from play.tailwindcss.com examples ;

This one is my code ;
https://play.tailwindcss.com/VVFKTYEf1g

The original one is as follow ;
https://play.tailwindcss.com/HFFlGJcUJB

I want to use full width of the parent element but wasn't able to do it. @alex-krasikau , I appreciate your quick response!

**Comment by sonmezcem** [comment, marked as answer]
@alex-krasikau thank you for helping me fix the issue. The issue was I had to use `grow` class and remove `md:w-2/3` and it uses full width now.

**Reply by alex-krasikau** [reply]
It always helps to try out in an isolated environment 🤘.
